src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -271,6 +274,52 @@ export default function formReducer(state = initialState, action) {
         data: { ...state.data, facilityType: action.facilityType },
       };
     }
+    case FORM_PAGE_FACILITY_V2_OPEN: {
+      return {
+        ...state,
+        childFacilitiesStatus: FETCH_STATUS.loading,
+      };
+    }
+    case FORM_PAGE_FACILITY_V2_OPEN_SUCCEEDED: {
+      let newSchema = action.schema;
+      const facilities = action.facilities;
+      // newSchema = unset('properties.vaFacilityMessage', newSchema);
+      newSchema = set(
+        'properties.vaFacility',
+        {
+          type: 'string',
+          enum: facilities.map(facility => facility.id),
+          enumNames: facilities.map(
+            facility =>
+              `${facility.name} (${facility.address?.city}, ${
+                facility.address?.state
+              })`,
+          ),
+        },
+        newSchema,
+      );
```

Comment:
Updates the enumOptions for the facilities radio widget with the fetched child facilities